src/bin/ch-remote.rs

```diff
@@ -445,14 +445,14 @@ fn rest_api_do_command(matches: &ArgMatches, socket: &mut UnixStream) -> ApiResu
                 .map_err(Error::HttpApiClient)
         }
         Some("restore") => {
-            let restore_config = restore_config(
+            let (restore_config, net_fds) = restore_config(
```

Comment:
Please have the changes to `ch-remote` in a separate commit.

Reply:
Moved the changes to a separate commit.